### DISCUSSION TITLE
Cleanup of rancher/rke cluster

### DISCUSSION THREAD

**Discussion by xhejtman** [rancher/rke2#4097] [Q&A]
Hello,

is there a way how to recognize currently used crds from rancher and which are obsolete and can be removed? We have a cluster running for almost 2 years, first version of rancher 2.6.0 has been installed and there seem to be some cdrs and possibly other objects that are not used anymore as of rke 1.24 and rancher 2.7.1. 

I do not think we use, e.g.:
```
vmwarevsphereconfigs.rke-machine-config.cattle.io               2021-11-07T12:02:56Z
vmwarevspheremachines.rke-machine.cattle.io                     2021-11-07T12:02:56Z
vmwarevspheremachinetemplates.rke-machine.cattle.io             2021-11-07T12:02:57Z
```

so they can be possibly removed but how to know it it breaks anything? This is clear case that wmware most probably does not break anything as we do not use vmware clusters.

**Comment by brandond** [comment]
This is a Rancher question, not an RKE2 question. I would transfer this to rancher/rancher but that repo does not have discussions enabled.
